ratpack/pom.xml

```diff
@@ -26,11 +26,21 @@
             <artifactId>ratpack-core</artifactId>
             <version>${ratpack.version}</version>
         </dependency>
+        <dependency>
+    		<groupId>org.codehaus.groovy</groupId>
+		    <artifactId>groovy-sql</artifactId>
+		    <version>2.4.15</version>
```

Comment:
I think these `version` number need to be in `properties`

Reply:
will make the change